deploy/helm/flightctl/values.kind.yaml

```diff
@@ -15,6 +15,7 @@ api:
     api: 3443 # this is also mapped in /test/scripts/kind_cluster.yaml as an extraPortMapping
     agent: 7443 # this is also mapped in /test/scripts/kind_cluster.yaml as an extraPortMapping
     grpc: 7444 # this is also mapped in /test/scripts/kind_cluster.yaml as an extraPortMapping
+    prometheus: 15690
```

Comment:
this value should not be here this is for the API server only

probably 

prometheus:
  nodePort: 15690

Reply:
aah no, ignore me, this is to make this available to metrics.